apps/dashboard/src/app/(dashboard)/(chain)/[chain_id]/[contractAddress]/_utils/getContractPageSidebarLinks.ts

```diff
@@ -0,0 +1,146 @@
+import type { SidebarLink } from "@/components/blocks/Sidebar";
+import type { ContractPageMetadata } from "./getContractPageMetadata";
+
+// todo - separate out "extensions" under extension group and not all links under same group
+
+export function getContractPageSidebarLinks(data: {
+  metadata: ContractPageMetadata;
+  contractAddress: string;
+  chainSlug: string;
+}) {
+  const layoutPrefix = `/${data.chainSlug}/${data.contractAddress}`;
+
+  const sidebarLinks: SidebarLink[] = [
+    {
+      group: "General",
+      links: [
+        {
+          label: "Overview",
+          href: layoutPrefix,
+          exactMatch: true,
```

Comment:
looks like this is `true` for all of them? can we remove it?

Reply:
It's not `true` for /nfts because ( it should be true for /nfts and /nfts/<token> )